pom.xml

```diff
@@ -60,6 +72,11 @@
             <artifactId>selenium-remote-driver</artifactId>
             <version>2.53.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>6.9.10</version>
+        </dependency>
     </dependencies>
 
```

Comment:
Move dependencies version to separate properties:
  <properties>
        <selenium-java.version>2.53.1</selenium-java.version>
  </properties>

and use those properties then:

        <dependency>
            <groupId>org.seleniumhq.selenium</groupId>
            <artifactId>selenium-java</artifactId>
            <version>${selenium-java.version}</version>
        </dependency>